lib/hammer_cli_foreman_discovery/discovery.rb

```diff
@@ -53,6 +53,12 @@ def send_request
         HammerCLIForeman::Fact::ListCommand.unhash_facts(super)
       end
 
+      def request_params
+        params = super
+        params['host'] = params.delete('discovered_host_id')
```

Comment:
According to API docs I think this should have been
```ruby
params['host_id'] = params.delete('discovered_host_id')
```
otherwise it will effectively ignore the parameter print all facts.